ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/RequestHandler.java

```diff
@@ -76,14 +83,22 @@ public RequestHandler(
 
   public KsqlEntityList execute(
       final List<ParsedStatement> statements,
-      final Map<String, Object> propertyOverrides
+      final Map<String, Object> overrides
```

Comment:
nit: why the name change from `propertyOverrides` to `overrides`? `propertyOverrides` carries more information, and is also used throughout the rest of the codebase (including in `executeRunScript()` below in this same file).

Reply:
it went beyond 100 chars in other lines 😂 I can change it back, I just thought it made the code look ugly...

Reply:
Actually.. looks like after whatever other changes I made this is no longer the case.